src/repository/posts.repository.js

```diff
@@ -0,0 +1,78 @@
+const { createPoolCluster } = require('mysql');
+const { connection } = require('../config/mysql');
+const { auth,db } = require('../config/firebase');
+const Post = require('../entities/post.entity');
+const { InvalidEntity } = require('../exceptions/InvalidEntity.exception');
+const { createPlaceholderString } = require('../utils/mysql.utils');
+
+function createPost(post){
+    return new Promise((resolve, reject)=>{
+        connection.query(
+            `INSERT INTO post VALUES (${createPlaceholderString(9)})`,
+            [
+                post.id,
+                post.user_id,
+                post.description,
+                post.services,
+                post.cleanliness,
+                post.taste,
+                post.price,
+                post.createdDate,
+                post.lastModifiedDate,
+            ], 
+            (error, results, fields) => {
+                error? reject(error):resolve(results);
+            }
+        )
+    });
+}
+
+function readPosts(user_id){
```

Comment:
readPosts is read the posts that were created by me?
or is read the posts that were created by my followings?